Objects/genericaliasobject.c

```diff
@@ -425,8 +456,37 @@ _Py_subs_parameters(PyObject *self, PyObject *args, PyObject *parameters, PyObje
             varparam = i;
         }
     }
+    PyObject *fillarg = NULL;
+    Py_ssize_t vartuplearg = nitems;
+    Py_ssize_t left = varparam;
+    Py_ssize_t right = nparams - varparam - 1;
     if (varparam < nparams) {
-        if (nitems < nparams - 1) {
+        for (Py_ssize_t i = 0; i < nitems; i++) {
+            PyObject *arg = _get_unpacked_var_tuple_arg(argitems[i]);
+            if (arg) {
+                if (vartuplearg < nitems) {
+                    Py_DECREF(arg);
+                    Py_DECREF(fillarg);
```

Comment:
This could use an `assert(fillarg)` to make sure it's not NULL.

Reply:
`Py_DECREF()` makes sure that the argument is not NULL. The result is the same -- a crash.